test/yaml.go

```diff
@@ -119,3 +120,83 @@ func GetKogitoServerlessOperatorBuilderConfig(path, namespace string) *corev1.Co
 	cm.Namespace = namespace
 	return cm
 }
+
+func GetPathForSamples(path string) string {
+	operatorPath := strings.Split(path, "kogito-serverless-operator")[1]
+	packages := strings.Count(operatorPath, "/")
+	if strings.Contains(operatorPath, "/test/") || packages == 3 {
```

Comment:
Not required, we can do it later. But maybe the `filepath` package might help here? https://gobyexample.com/file-paths

Reply:
We do later, is a func used only in test